### ISSUE TITLE
Installation of dm-acme[envs] fails because of bsuite dependency not pointing to PyPI.

### ISSUE THREAD

**Issue by redfungus** [deepmind/acme#98]
Installation fails with the following error:

```
> [2021-02-20T17:06:14.776Z] [PID 11852] ERROR: Packages installed from PyPI cannot depend on packages which are not also
>  hosted on PyPI.
> dm-acme depends on bsuite@ git+git://github.com/deepmind/bsuite.git#egg=bsuite
```

This is due to the dependency of `bsuite` in `env_requirements` pointing to the GitHub repo instead of the PyPI package. 
See here: [setup.py](https://github.com/deepmind/acme/blob/master/setup.py#52)

The package `rlax` will potentially cause a similar error for `jax_requirements`. 
See here: [setup.py](https://github.com/deepmind/acme/blob/master/setup.py#46)

**Comment by cool-RR** [comment]
I am also affected by this problem.

**Comment by fastturtle** [comment]
@redfungus do you want to submit a PR to fix the issue with rlax as well?

**Comment by redfungus** [comment]
@fastturtle Yes I will do that very soon
I had mistakenly thought that rlax is not published on PyPI thats why I didn't include it in the first PR

**Comment by redfungus** [comment]
@fastturtle I'm not sure why the tests are failing. Any ideas?

**Comment by fastturtle** [comment]
@redfungus it's because rlax's master branch includes some breaking changes that have not made it into their pypi package yet. I'm discussing with the rlax developers and they will push a new release to pypi that we can use :)

**Comment by 8bitmp3** [comment]
@fastturtle Any update would be appreciated. 

Similar issue: installing Acme with TensorFlow (`dm-acme[tf]`) and Acme with envs (`dm-acme[envs]`) worked fine with `bsuite` "operational". However, running `pip install dm-acme[jax]` ended with a similar error:

```
ERROR: Packages installed from PyPI cannot depend on packages which are not also hosted on PyPI.
dm-acme depends on rlax@ git+git://github.com/deepmind/rlax.git#egg=rlax 
```

Manually installing Rlax via `github.com/deepmind/rlax` and then running `pip install dm-acme[jax]` again seems to resolve this.

**Comment by mwhoffman** [comment]
Hi @redfungus and @8bitmp3 this should be fixed on github as of commit 0dda5efed4cf2b2592441357352524ff4638d192. The rlax version was needed for some functionality that had not been pushed to pypi as well, but that should all be sorted out now as of 0.0.4. Finally, we should be pushing a new revision to pypi soon, which will then correct this on pip. I'll close this in the meantime as it should be fixed by the next release (or if you pull directly from GH).

**Comment by siggiorn** [comment]
I'm experiencing a similar issue here: https://github.com/deepmind/acme/issues/150
